packages/migrate/tests/index.test.ts

```diff
@@ -21,21 +21,21 @@ describe('migrate', () => {
   function clean(dir: string): void {
     fs.rmSync(dir, { recursive: true, force: true });
   }
-
+  // copy the fixture files to a tmp directory
   function prepareInputFiles(files: string[] = ['index.js']): string[] {
```

Comment:
We have helper in utils https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/plugins/test/re-rehearse.plugin.test.ts#L18

Reply:
https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/utils/src/tests.utils.ts